specification/sql/resource-manager/readme.md

```diff
@@ -693,6 +693,26 @@ override-info:
 
 The following packages are each composed of all apis from only one api-version.
 
+### Tag: package-pure-2020-11-preview
```

Comment:
we have already had the 2020-11 tag in line 35. maybe you can move them to here.

Reply:
ack